spec/rails_helper.rb

```diff
@@ -11,6 +11,8 @@
 require File.join(File.dirname(__FILE__), 'support/database_cleaner')
 require 'database_cleaner'
 require 'capybara/rspec'
+require 'capybara/poltergeist'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.